exchanges/order/orders.go

```diff
@@ -7,11 +7,13 @@ import (
 	"strings"
 	"time"
 
+	"github.com/thrasher-corp/gocryptotrader/common"
 	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/validate"
 )
 
 // Validate checks the supplied data and returns whether or not it's valid
-func (s *Submit) Validate() error {
+func (s *Submit) Validate(opt ...validate.Checker) error {
 	if s == nil {
 		return ErrSubmissionIsNil
 	}
```

Comment:
Here's a conceptual bug 🎉 Bitfinex orders need to be negative in order to sell. So this validate function prevents that. I'd suggest a fix inside the Bitfinex wrapper which is after the validate function: `if o.Side == order.Sell { o.Amount *= -1 }`

Reply:
Just re-read our code and the documentation. We use Bitfinex v1 to submit orders, which doesn't have the negative amount to indicate sell orders. So don't worry about this nit 😆